internal/schema.go

```diff
@@ -141,6 +141,23 @@ type IDPEntityDescriptor struct {
 	ASSERTION        string           `xml:"xmlns:assertion,attr"`
 	EntityId         string           `xml:"entityID,attr"`
 	IDPSSODescriptor IDPSSODescriptor `xml:"IDPSSODescriptor"`
+	Organization Organization `xml:"Organization,omitempty "`
```

Comment:
@sccalabr please take Organization type to *Organization , so if Organization is not set, this will not render in the metadata xml.

Reply:
Does Contact Person also need the star? Or no because it is an array?